modular_skyrat/modules/blueshield/code/blueshield.dm

```diff
@@ -64,7 +63,8 @@
 	box = /obj/item/storage/box/survival/security
 	belt = /obj/item/modular_computer/pda/security
 
-	id_trim = /datum/id_trim/job/blueshield
+	id = /obj/item/card/id/advanced/silver //BUBBER EDIT original: obj/item/card/id/advanced/CENTCOM
+	id_trim = /datum/id_trim/job/blueshield //BUBBER EDIT
```

Comment:
What trim were they previously using?

Reply:
You know, now that I think about it, it's not a bubber edit. It's the same id_trim.